.github/workflows/release.yml

```diff
@@ -34,7 +34,24 @@ jobs:
         run: sudo apt-get update && sudo apt-get install -y pandoc
 
       - name: Create release
-        run: ~/auto shipit
+        run: |
+          new_version="$(~/auto shipit --dry-run --quiet)"
+          if [ -z "$new_version" ]
+          then echo "Nothing to release"
+               exit
+          fi
+          changelog_version="$(perl -nle '/^# ([0-9]+\.\S*) \(.*\)/ and print $1 and exit' CHANGELOG.md)"
+          opts=
+          if dpkg --compare-versions "$new_version" lt "$changelog_version"
+          then echo "[ERROR] New version is less than latest version in changelog"
+               exit 1
+          elif dpkg --compare-versions "$new_version" eq "$changelog_version"
+          then opts=--no-changelog
+          elif ! dpkg --compare-versions "$new_version" gt "$changelog_version"
+          then echo "[ERROR] Could not compare new version '$new_version' to changelog version '$changelog_version'"
+               exit 1
+          fi
```

Comment:
an idea!  @jwodder -- would it be possible to move this checking into a separate step, make this workflow run also on PRs which have `release` label, but without actually doing `~/auto shipit`?
Then we could be certain that `release` PR is a kosher one.
Workflow should react on change to labels (like the other one).

Reply:
@yarikoptic No, if the workflow is running on an open `release` PR, `auto` won't consider the project eligible for a release (as the PR isn't merged yet), and so `auto shipit --dry-run --quiet` will output nothing.

Reply:
damn, it was a good idea! ;)  just to make sure -- it wouldn't work even if we adjust `"baseBranch": "maint"` to point to current to PR merge branch (I guess there is none on github) in `.autorc`, right?

Reply:
@yarikoptic I *think* that wouldn't work, as then `auto` wouldn't pick up the PRs that had already been merged into `maint`, and so the version bump level would be potentially inaccurate.